code/ai/aicode.cpp

```diff
@@ -13681,7 +13678,7 @@ int aas_1(object *objp, ai_info *aip, vec3d *safe_pos)
 		float outer_rad = ship_get_exp_outer_rad(ship_objp);
 
 		if (vm_vec_dist_quick(&objp->pos, &ship_objp->pos) > outer_rad*1.5f) {
-			aip->ai_flags &= ~AIF_AVOID_SHOCKWAVE_WEAPON;
+			aip->ai_flags.remove(AI::AI_Flags::Avoid_shockwave_ship);
```

Comment:
This is (technically!) a behaviour change. Do note however that the original behaviour was wrong: At this point in the code, we're dealing with the question of whether or not to avoid a shockwave from a ship, not a weapon. Some code archeology is needed whether this is a change introduced by the SCP, or whether it was present in the original source release.

I also think the question of whether or not we need two Avoid_shockwave flags is absolutely necessary should be investigated.